File: Selenium/Activities/Activity11.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

import java.io.IOException;

public class Activity11 {
    public static void main(String[] args){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get(" https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("Title of webpage:" + driver.getTitle());
        WebElement checkBox=driver.findElement(By.xpath("//input[@class='willDisappear']"));
        checkBox.click();
        boolean checkSelected=checkBox.isSelected();
        System.out.println("Checkbox selected="+checkSelected);
        checkBox.click();
        checkSelected=checkBox.isSelected();
        System.out.println("Checkbox selected="+checkSelected);
        driver.close();
    }
}

File: Selenium/Activities/Activity11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    wait = WebDriverWait(driver, 10)
    print("Title of webpage:" + driver.title)
    checkBox = driver.find_element(By.XPATH, "//input[@class='willDisappear']")
    checkBox.click()
    checkSelected = checkBox.is_selected()
    print("Checkbox selected=" + str(checkSelected))
    checkBox.click()
    checkSelected = checkBox.is_selected()
    print("Checkbox selected=" + str(checkSelected))
    driver.close()
